### ISSUE TITLE
Use font icons for icons to be aligned with other elements

### ISSUE THREAD

**Issue by tomivirkki** [vaadin/vaadin-time-picker#18]
...once it's ready design-wise

**Comment by jouni** [comment]
I remember a discussion about not adding any font icons to the core styles (although some of the components currently have them).

For Lumo, the icon(s) should be part of `vaadin-lumo-styles`.

**Comment by manolo** [comment]
Should we have a ticket with the contrary task in those elements instead: remove icons and move to lumo ?

**Comment by jouni** [comment]
Perhaps. Though, no need to move anything to Lumo, it already has all the icons needed (the icons used by the core styles are not the same).

**Comment by manolo** [comment]
So in this case  the simplest option might be replace iron-icon with to have a unicode character in the core css `content` property, it can be easily themable in lumo then. 
The same could be done in others, e.g in date-picker we would save ~ 2.5Kb in core

**Comment by jouni** [comment]
Are we using iron-icon in any of the components?

**Comment by manolo** [comment]
Yes, in this component. https://github.com/vaadin/vaadin-time-picker/blob/master/src/vaadin-time-picker.html#L51
This issue is precisely about replacing it.
As I mentioned I think the simplest solution for resolving this ticket is an unicode character in core, then themes can change font-family and content. Then the same solution might be applied to current implementation in other elements replacing the custom font icons in their cores with unicode, and saving their core size

**Comment by tomivirkki** [comment]
The icon for MD https://material.io/tools/icons/?search=time&icon=access_time&style=baseline
